at-spi2-core 2.50.0

cho-m opened the pull request:
<!-- Use [x] to mark item done, or just click the checkboxes with device pointer -->

- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

chenrui333 commented:
We can probably also deprecate atk as well (atk has been merged into at-spi2-core, https://github.com/Homebrew/homebrew-core/pull/111065)

cho-m commented:
> We can probably also deprecate atk as well (atk has been merged into at-spi2-core, #111065)

I just removed it and added to formula_renames. I think that should allow `brew upgrade` to handle switching users to `at-spi2-core` better.

chenrui333 commented:
Great work in here! :100
